packages/web-components/fast-components/src/tooltip/tooltip.stories.ts

```diff
@@ -25,12 +26,30 @@ function onAnchorMouseEnter(e: MouseEvent): void {
     tooltipInstance.anchorElement = e.target as HTMLElement;
 }
 
+//check changing anchor by setting attribute
+function onAnchorMouseEnterAttribute(e: MouseEvent): void {
+    if (!e.target) {
+        return;
+    }
+
+    const tooltipInstance = document.getElementById(
+        "tooltip-anchor-switch"
+    ) as FoundationTooltip;
+    tooltipInstance.setAttribute("anchor", (e.target as HTMLElement).id);
+}
+
 addons.getChannel().addListener(STORY_RENDERED, (name: string) => {
     if (name.toLowerCase().startsWith("tooltip")) {
         document
-            .querySelectorAll("fast-button[id^=anchor-anchor-switch]")
+            .querySelectorAll("fast-button[id^=anchor-anchor-switch-prop]")
```

Comment:
Why use ids here instead of classes?

Reply:
tooltips/anchors are associated by passing id's so that seemed the logical way to identify elements in the sample